nixos/modules/system/boot/resolved.nix

```diff
@@ -205,11 +205,12 @@ in
 
       boot.initrd.systemd = {
         contents = {
-          "/etc/tmpfiles.d/resolv.conf".text =
-            "L /etc/resolv.conf - - - - /run/systemd/resolve/stub-resolv.conf";
           "/etc/systemd/resolved.conf".text = resolvedConf;
         };
 
+        tmpfiles.settings.resolvconf."/etc/resolv.conf".L.argument =
```

Comment:
the name could be confusing because of resolvconf actually exists and is a different file

Reply:
Ok, renamed.